packages/components/input/src/InputLeadingAddon.tsx

```diff
@@ -7,7 +7,11 @@ export type InputLeadingAddonProps = InputAddonProps
 
 export const InputLeadingAddon = forwardRef<HTMLDivElement, InputLeadingAddonProps>(
   ({ className, ...others }, ref) => (
-    <InputAddon ref={ref} className={cx(className, 'rounded-l-lg border-r-none')} {...others} />
+    <InputAddon
+      ref={ref}
+      className={cx(className, 'rounded-l-lg !rounded-r-none mr-[-1px]')}
```

Comment:
I noticed you're using a negative margin of 1px here. Have you tested to make sure it's reliable and won't break when zooming in or out, stuff like that?

Reply:
This negative margin is to make the right shadow of the input overlap with the left shadow of the addon. That way when focus goes from the input to the input it looks seamless. 

It works well when zooming in too.